### DISCUSSION TITLE
Math php and Latex

### DISCUSSION THREAD

**Discussion by GeO95** [markrogoyski/math-php#442] [Q&A]
Hello there
I’ve read the readme but I still have a question: is there a way this library can accept Latex inputs? And then work on it.
Thanks in advance

**Comment by Beakerboy** [comment]
I imagine this would require a language parser as well as some sort of generic “mathematical expression” class or family of classes. MathPHP currently has no string to expression conversion of any sort.

If this is a feature you would like to work on, we welcome all types of feature contributions.
